controllers/comments_controller.js

```diff
@@ -12,6 +12,16 @@ exports.getCommentsByArticleId = async (req, res, next) => {
         next(err)
     }
 }
+exports.removeCommentById = async (req, res, next) => {
+    try {
+        const { comment_id } = req.params
+        await selectCommentById(comment_id)
```

Comment:
I think PSQL should throw an error anyway if it doesn't find a comment by commentID so you wouldn't need a seperate check.